charts/konk/templates/configmap.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "konk.fullname" . }}-configmap
+  labels:
+    app: {{ include "konk.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+  start.sh: |
+    kubeadm init phase certs all
+    kubeadm init phase kubeconfig admin
+    VAL=$(kubectl -n {{ .Release.Namespace }} get secret {{ include "konk.fullname" . }}-etcd-cert --ignore-not-found)
+    if [[ $VAL == "" ]]
+    then
+      kubectl -n {{ .Release.Namespace }} create secret generic {{ include "konk.fullname" . }}-etcd-cert --from-file=/etc/kubernetes/pki/etcd/ca.crt --from-file=/etc/kubernetes/pki/etcd/server.crt --from-file=/etc/kubernetes/pki/etcd/server.key --from-file=/etc/kubernetes/pki/apiserver-etcd-client.crt --from-file=/etc/kubernetes/pki/apiserver-etcd-client.key
+    fi
+    VAL=$(kubectl -n {{ .Release.Namespace }} get secret {{ include "konk.fullname" . }}-kubeconfig --ignore-not-found)
+    if [[ $VAL == "" ]]
+    then
+      kubectl -n {{ .Release.Namespace }} create secret generic {{ include "konk.fullname" . }}-kubeconfig --from-file=/etc/kubernetes/admin.conf
+    fi
```

Comment:
Please move the bash code into charts/konk/files/start.sh (or maybe provision.sh) and load it into the ConfigMap like this: https://helm.sh/docs/chart_template_guide/accessing_files/#configmap-and-secrets-utility-functions. This way we can use the bash script outside of the helm chart if we want, like for local development.

Reply:
Moved the bash script to provision.sh under the scripts directory.